src/Rings/mpoly-localizations.jl

```diff
@@ -249,7 +249,7 @@ function prime_ideal(S::MPolyComplementOfKPointIdeal)
     m = ideal(R, [x[i]-a for (i, a) in enumerate(point_coordinates(S))])
     set_attribute!(m, :is_prime=>true)
     kk isa Field && set_attribute!(m, :is_maximal=>true)
-    set_attribute!(m, :dim=>dim(coefficient_ring(ring(S))))
+    set_attribute!(m, :krull_dim=>krull_dim(coefficient_ring(ring(S))))
```

Comment:
Here the attribute name is fixed -- at least I think that's the correct thing to do here?